mesop/server/server.py

```diff
@@ -44,10 +43,17 @@
 def configure_flask_app(
   *, prod_mode: bool = True, exceptions_to_propagate: Sequence[type] = ()
 ) -> Flask:
+  static_folder = get_static_folder()
+  static_url_path = get_static_url_path()
+  if static_folder and static_url_path:
+    print(f"Static folder enabled: {static_folder}")
+  else:
```

Comment:
imo, I think it's a little spammy to print this out (in the disabled case). we should eventually move most/if not all of our print statements into proper logger statements #903, but for now we should avoid adding prints in the base/default cause.